### PR TITLE
RHDEVDOCS-3296 - Add a link to OL 5.1.2 advisory to 5.1 release notes.

### PR DESCRIPTION
**Pull request by libander**
- Aligned team: Dev Tools
- For branches: master, enterpris-4.8+
- https://issues.redhat.com/browse/RHDEVDOCS-3296
- Direct link to doc preview: https://deploy-preview-36379--osdocs.netlify.app
SME review: @periklis
QE review: @kabirbhartiRH
Peer review: @rolfedh 
- Please merge now.

### CODE REVIEW

**Review by periklis — COMMENTED** [review]
LGTM

File: logging/cluster-logging-release-notes.adoc [hunk 1]
```diff
@@ -20,6 +20,7 @@ Red Hat is committed to replacing problematic language in our code, documentatio
 [id="cluster-logging-release-notes-5-2-0"]
 == OpenShift Logging 5.2.0
 
+This release includes link:https://access.redhat.com/errata/RHBA-2021:3550[RHBA-2021:3550 OpenShift Logging Bug Fix Release 5.2.1].
```
Comment (rolfedh) [inline logging/cluster-logging-release-notes.adoc:23]:
```suggestion
* This release includes link:https://access.redhat.com/errata/RHBA-2021:3550[RHBA-2021:3550 OpenShift Logging Bug Fix Release 5.2.1].
```

File: logging/cluster-logging-release-notes.adoc [hunk 2]
```diff
@@ -20,6 +20,7 @@ Red Hat is committed to replacing problematic language in our code, documentatio
...
 This release includes link:https://access.redhat.com/errata/RHBA-2021:3393[RHBA-2021:3393 OpenShift Logging Bug Fix Release 5.2.0].
```
Comment (rolfedh) [inline logging/cluster-logging-release-notes.adoc:24]:
```suggestion
* This release includes link:https://access.redhat.com/errata/RHBA-2021:3393[RHBA-2021:3393 OpenShift Logging Bug Fix Release 5.2.0].
```

File: modules/cluster-logging-release-notes-5.1.0.adoc [hunk 3]
```diff
@@ -3,6 +3,7 @@
 
 This release includes:
```
Comment (rolfedh) [inline modules/cluster-logging-release-notes-5.1.0.adoc:4]: The following advisories are available for OpenShift Logging 5.2.x:

File: modules/cluster-logging-release-notes-5.1.0.adoc [hunk 4]
```diff
@@ -1,8 +1,9 @@
 [id="cluster-logging-release-notes-5-1-0"]
 = OpenShift Logging 5.1.0
 
-This release includes:
+The following advisories are available for OpenShift Logging 5.2.x:
```
Comment (periklis) [inline modules/cluster-logging-release-notes-5.1.0.adoc:4]: Should that be `The following advisories are available for OpenShift Logging 5.1.x`?

**Review by periklis — CHANGES_REQUESTED** [review]

**Review by rolfedh — APPROVED** [review]
@libander: After updating this PR for @periklis comment, please confirm that the advisory is live (the team decided to release the advisories today), and ask the merge-person to merge this PR.

**Review by periklis — APPROVED** [review]
/lgtm

**Pull request merged**